### PR TITLE
Document the new options available on the DocumentPictureInPicture.requestWindow() method

### PR DESCRIPTION
**Pull request by chrisdavidmills**
### Description

This PR adds documentation for two new options available to the [`DocumentPictureInPicture.requestWindow()`](https://developer.mozilla.org/en-US/docs/Web/API/DocumentPictureInPicture/requestWindow) method:

- Chrome 124 adds the `disallowReturnToOpener` option. When set to `true`, it hides the "back to tab" button that by default appears in the top bar of the PIP window. See https://chromestatus.com/feature/6223347936657408.
- Chrome 130 adds the `preferInitialWindowPlacement` option. When set to `true`, it causes the PIP window to always appear back at its default initial position and size when closed and then reopened. By default, it will reopen at its previous position and size. See https://chromestatus.com/feature/5183881532932096.

I've tested that both features work in the latest Chrome, but I've not tested exact versions in detail.

### CODE REVIEW

**Review by beaufortfrancois — COMMENTED** [review]
LGTM with existing comment

File: files/en-us/web/api/documentpictureinpicture/requestwindow/index.md [hunk 1]
```diff
@@ -25,14 +25,30 @@ requestWindow(options)
...
-    - `width`
-      - : A non-negative number representing the width to set for the Picture-in-Picture window's viewport, in pixels. If `options` is not specified, the default value 0 is used.
+
+    - `disallowReturnToOpener` {{optional_inline}}
+
+      - : A boolean value; defaults to `false`. When set to `true`, it hints to the browser that it should hide the "back to tab" button, illustrated in the below image, which by default appears in the top bar of the Picture-in-Picture window.
+
+        ![browser window containing an embedded video player and multiple control buttons, with a back to tab button in the top bar, highlighted with a red box](back-to-tab-button.png)
+
+        The "back to tab" button hides the window when pressed, returning focus to the spawning tab.
```
[the remark this answers is not in the document]
Comment (chrisdavidmills) [reply files/en-us/web/api/documentpictureinpicture/requestwindow/index.md]: Thanks @wbamberg. I agree with your comments — the text could have been more accurate, and it is not good to tie the description to a specific implementation. However, I still think it is useful to show what a particular example of the UI control could look like, to help the reader visualize this option's effect. Plus I spent the time making the frickin' screenshot ;-)

So, I have updated it to use your text, but I have kept the screenshot as an example of what the UI control looks like in a specific implementation, and described it clearly.

WDYT?

[the remark this answers is not in the document]
Comment (wbamberg) [reply files/en-us/web/api/documentpictureinpicture/requestwindow/index.md]: I still think omitting the screenshot is better but I can live with this. Thanks!

**Review by wbamberg — APPROVED** [review]
👍 thanks!

**Pull request merged**